RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+This release adds a setting to the public API, and does some internal cleanup:
+
+- The `~hypothesis.settings.derandomize` setting is now documented (:issue:`890`)
```

Comment:
IIRC without `:func:` the `~` would have no effect.

Reply:
Well spotted - in fact it doesn't format this at all!